lang/LangPrimSource/SC_ComPort.cpp

```diff
@@ -205,8 +205,25 @@ UDP::UDP(int inPortNum, HandlerType handlerType, int portsToCheck): mPortNum(inP
         }
     }
 
-    boost::asio::socket_base::send_buffer_size option(65536);
-    mUdpSocket.set_option(option);
+    boost::asio::socket_base::send_buffer_size send_buffer_size;
+    mUdpSocket.get_option(send_buffer_size);
+    if (send_buffer_size.value() < 8 * 1024 * 1024) {
+        send_buffer_size = 8 * 1024 * 1024;
+        try {
+            mUdpSocket.set_option(send_buffer_size);
+        } catch (boost::system::system_error& e) {}
+        mUdpSocket.get_option(send_buffer_size);
+    }
+
+    boost::asio::socket_base::receive_buffer_size recv_buffer_size;
+    mUdpSocket.get_option(recv_buffer_size);
+    if (recv_buffer_size.value() < 8 * 1024 * 1024) {
```

Comment:
Similarly, let's add
```
static constexpr int receiveBufferSize = 8 * 1024 * 1024;
```
to `UDP::UDP`.

(In the future, someone may want to use different buffer sizes for sending and receiving.)